code/parachain/frame/staking-rewards/src/benchmarking.rs

```diff
@@ -49,13 +51,20 @@ fn get_reward_pool<T: Config>(
 
 fn lock_config<T: Config>() -> LockConfig<T::MaxStakingDurationPresets> {
 	LockConfig {
-		duration_presets: [
-			(ONE_HOUR, FixedU64::from_rational(101, 100).try_into_validated().expect(">= 1")), /* 1% */
-			(ONE_MINUTE, FixedU64::from_rational(1_001, 1_000).try_into_validated().expect(">= 1")), /* 0.1% */
-		]
-		.into_iter()
-		.try_collect()
-		.unwrap(),
+		duration_multipliers: DurationMultipliers::Presets(
+			[
+				// 1%
+				(ONE_HOUR, FixedU64::from_rational(101, 100).try_into_validated().expect(">= 1")),
+				// 0.1%
+				(
+					ONE_MINUTE,
+					FixedU64::from_rational(1_001, 1_000).try_into_validated().expect(">= 1"),
+				),
+			]
+			.into_iter()
+			.try_collect()
+			.unwrap(),
```

Comment:
maybe also use bounded_btree_map! as in test/mod.rs?

Reply:
not possible since that macro isn't available in no_std (which benchmarks are)